src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -401,8 +402,15 @@ const BaseCourseCard = ({
   };
 
   const renderStartDate = () => {
-    const formattedStartDate = startDate ? dayjs(startDate).format('MMMM Do, YYYY') : null;
-    const isCourseStarted = dayjs(startDate) <= dayjs();
+    // TODO: Determine if its worth exposing weeks_to_complete in assignments to utilize this function effectively
```

Comment:
nit: similar to other comment, include a ticket link here knowing we do want to have this code path be parity with others.